package.json

```diff
@@ -4,7 +4,7 @@
   "description": "React Google Analytics Module.",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha test/*.test.js && gulp test",
+    "test": "mocha test/*.test.js && mocha test/components/*.test.js && gulp test",
     "build": "gulp build"
```

Comment:
i tried `mocha test/**/*.test.js` but somehow `test/index.js` didn't get run (i don't know why) :confounded: ... that's why I had to change it to `mocha test/*.test.js && mocha test/components/*.test.js` instead.